Makefile.inc

```diff
@@ -32,6 +34,10 @@ endif
 LDFLAGS     = $(FCFLAGS)
 LIBS        = -L${SCALAPACK_PATH}/lib -lscalapack -lreflapack -lrefblas
 
+ifneq ($(CINT),)
+FCDEBFLAGS := $(subst -std=f2003,-std=f2008ts,$(FCDEBFLAGS))
```

Comment:
btw, the GCC devs made it easier and deprecated the f2008ts flag (now part of f2008)

Reply:
if it so, then good. There was some error which was solved when I changed to the f2008ts format.

Reply:
I guess we have to keep f2008ts for the moment, they changed it for gcc >= 7, sorry